src/NHSD.BuyingCatalogue.Identity.Api/appsettings.Development.json

```diff
@@ -46,12 +49,5 @@
       "resourceName": "Organisation",
       "displayName": "Organisation"
     }
-  ],
-  "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft": "Warning",
-      "Microsoft.Hosting.Lifetime": "Information"
-    }
-  }
+  ]
```

Comment:
Why have you removed the logging section?

Reply:
Because we're using Serilog and all the configuration is done in code in `Program.Main` based on the [advice](https://nblumhardt.com/2019/10/serilog-in-aspnetcore-3/) of the author of Serilog. Therefore, the logging section isn't required.